htdocs/comm/propal/list.php

```diff
@@ -2321,17 +2321,17 @@
 			}
 			// Note public
 			if (!empty($arrayfields['p.note_public']['checked'])) {
-				print '<td class="center">';
-				print dol_string_nohtmltag($obj->note_public);
+				print '<td class="sensiblehtmlcontent center">';
+				print dol_string_onlythesehtmltags($obj->note_public);
```

Comment:
With v19, there is a new method
dolPrintHtml... 
Can you try with it ?

Reply:
cool, i did it @eldy